process/factory/shard/interceptorsContainerFactory.go

```diff
@@ -630,8 +629,25 @@ func (icf *interceptorsContainerFactory) generateTrieNodesInterceptors() ([]stri
 	keys = append(keys, identifierTrieNodes)
 	interceptorSlice = append(interceptorSlice, interceptor)
 
-	identifierTrieNodes = factory.TrieNodesTopic + shardC.CommunicationIdentifier(sharding.MetachainShardId)
+	identifierTrieNodes = factory.ValidatorTrieNodesTopic + shardC.CommunicationIdentifier(shardC.SelfId())
```

Comment:
There are tries for validators on the shards as well? Not just the metashard?

Reply:
no tries for validators on shards.